content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -427,7 +355,7 @@ spec:
 
 The field `someRandomField` has been pruned.
 
-Note that the `kubectl create` call uses `--validate=false` to skip client-side validation. Because the [OpenAPI validation schemas are also published](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#publish-validation-schema-in-openapi-v2) to kubectl, it will also check for unknown fields and reject those objects long before they are sent to the API server.
+Notice that the `kubectl create` call uses `--validate=false` to skip client-side validation. Because the [OpenAPI validation schemas are also published](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#publish-validation-schema-in-openapi-v2) to kubectl, it will also check for unknown fields and reject those objects long before they are sent to the API server.
```

Comment:
This seems confusing. Is this correct?

To skip client-side validation, use `kubectl create --validate=false`.
Besides client validation, `kubectl` uses the  [published OpenAPI validation schema](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#publish-validation-schema-in-openapi-v2) to check for unknown fields and reject those objects before they are sent to the API server.

Reply:
I think it'd be best to fix that line outside of the scope of this PR (title: “Document CustomResourceDefinition as stable”)